src/highdicom/pr/sop.py

```diff
@@ -0,0 +1,1542 @@
+            ref_im_rows
+        ]
+        display_area_item.PresentationSizeMode = 'SCALE TO FIT'
+        display_area_item.PresentationPixelAspectRatio = [1, 1]
+        self.DisplayedAreaSelectionSequence = [display_area_item]
+
+        # Modality LUT
+        if sop_class_uid in (
+            GrayscaleSoftcopyPresentationStateStorage,
+            PseudoColorSoftcopyPresentationState,
+        ):
+            if copy_modality_lut:
+                if (
+                    rescale_intercept is not None or
+                    rescale_slope is not None or
+                    rescale_type is not None
+                ):
+                    raise TypeError(
+                        'If argument "copy_modality_lut" is  True, arguments '
+                        '"rescale_slope", "rescale_intercept", and '
+                        '"rescale_type" must all be None.'
+                    )
+                self._copy_modality_lut(referenced_images)
+            else:
+                self._add_modality_lut(
+                    rescale_intercept=rescale_intercept,
+                    rescale_slope=rescale_slope,
+                    rescale_type=rescale_type,
+                    modality_lut=modality_lut
+                )
+
+        # Softcopy VOI LUT
+        if copy_voi_lut:
+            if softcopy_voi_luts is not None:
+                raise TypeError(
+                    'If argument "copy_voi_lut" is  True, argument '
+                    '"softcopy_voi_luts" must be None.'
+                )
+            self._copy_voi_lut(referenced_images)
+        elif softcopy_voi_luts is not None:
+            if len(softcopy_voi_luts) == 0:
+                raise ValueError(
+                    'Argument "softcopy_voi_luts" must not be empty.'
+                )
+            for v in softcopy_voi_luts:
+                if not isinstance(v, SoftcopyVOILUT):
+                    raise TypeError(
+                        'Items of "softcopy_voi_luts" must be of type '
+                        'highdicom.pr.SoftcopyVOILUT.'
+                    )
+
+                # If the softcopy VOI LUT references specific images,
+                # check that the references are valid
+                if hasattr(v, 'ReferencedImageSequence'):
+                    for item in v.ReferencedImageSequence:
+                        uids = (
+                            item.ReferencedSOPClassUID,
+                            item.ReferencedSOPInstanceUID
+                        )
+                        if uids not in ref_uids:
+                            raise ValueError(
+                                f'Instance with SOP Instance UID {uids[1]} and '
+                                f'SOP Class UID {uids[0]} is referenced in '
+                                'items of "softcopy_voi_luts", but not '
+                                'included in "referenced_images".'
+                            )
+
+            self.SoftcopyVOILUTSequence = softcopy_voi_luts
```

Comment:
In case of the Blending Softcopy Presentation State IOD, the Softcopy VOI LUT Sequence is located under the Blending Sequence instead of the root.

This is another reason for not using inheritance, but using setter functions instead. See [this comment](https://github.com/herrmannlab/highdicom/pull/139/files#r841299418)

Reply:
See 990bf06b176612fc92bbf580d7e681a486a32bc2